backend/FwLite/MiniLcm.Tests/NormalizationTests.cs

```diff
@@ -0,0 +1,87 @@
+using MiniLcm.Validators;
+using Moq;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace MiniLcm.Tests;
+
+public class NormalizationTests
+{
+    public IMiniLcmApi MockApi { get; init; }
+    public IMiniLcmApi NormalizingApi { get; init; }
+
+    public const string NFCString = "naïve"; // U+00EF LATIN SMALL LETTER I WITH DIAERESIS
+    public const string NFDString = "naïve"; // U+0069 LATIN SMALL LETTER I + U+0308 COMBINING DIAERESIS
```

Comment:
I'm worried that these might get messed up at some point. What if we just explicitly call `Normlaize` on them at runtime instead so we can be sure what they are. Either that or we assert on creation that they are what we expect.

Reply:
Fixed in commit 2a5e6fc2b: by changing the consts to use `\u00ef` escapes, we can ensure that they don't accidentally get messed up by some IDE automatically normalizing the source file.

Reply:
That's commit 876384c now after the rebase.